hack/lokistack_dev.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: loki.openshift.io/v1beta1
+kind: LokiStack
+metadata:
+  name: lokistack-dev
+spec:
+  size: 1x.extra-small
+  replicationFactor: 2
```

Comment:
The replication factor for `1x.extra-small` can be only 1 for now as it supports a single ingester instance. In turn this helps to restart the ingester faster when testing dev changes.